README.md

```diff
@@ -1 +1,55 @@
-# Mobile_First
\ No newline at end of file
+# Mobile_First
+
+> The project contanin some data for your review. Data contains mobile first setup which makes the project handy for use.
+
+
+## Built With
+
+- Html
+- Css used
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+Browser and git
+
+### Install
+git clone https://github.com/Alabi12/helloMicroverse.git
+
+### Usage
+open in browser
+
```

Comment:
- [ ] [OPTIONAL] Good job so far 👏. In addition to what you have done, I would suggest that you edit this section I other to look more professional. In this section, you are expected to describe procedures to clone and use your project locally. Please add the procedures in a very good way 😃